### ISSUE TITLE
RRD file is too small (camera module)

### ISSUE THREAD

**Issue by ma-zal** [dgomes/rrd_recorder#10]
After Home Assistant upgrade to 2021.8.6 i see errors "Some File is too small". It is happening in all my RRD files. The expected file size is exactly 8 bytes higher than reality.

Logs:

```
ERROR (SyncWorker_0) [custom_components.rrd.camera] '/rrd/power.rrd' is too small (should be 23852 bytes)
ERROR (SyncWorker_1) [custom_components.rrd.camera] '/rrd/power.rrd' is too small (should be 23852 bytes)
ERROR (SyncWorker_3) [custom_components.rrd.camera] '/rrd/pow43.rrd' is too small (should be 38440 bytes)
ERROR (SyncWorker_5) [custom_components.rrd.camera] '/rrd/temp_out.rrd' is too small (should be 124892 bytes)
ERROR (MainThread) [homeassistant.components.camera] Error while setting up rrd platform for camera
```

Files are correct, no corruption. I tried to use `rrd dump` and restore dumped xml back to rrd by `rrd restore`, but it did not solve the issue.

It looks like the error message is not from this `rrd_recorder`, but it comes from Python `rrdtool` lib and it happens during command ` rrdinfo = rrdtool.info(rrd_file)` in `camera.py`.

Note: My Home Assistant runs in docker on Rasperry and it is from official Docker image `homeassistant/raspberrypi4-homeassistant:stable`.

--
I have totally no idea, what is wrong.  Anybody facing same issue?

**Comment by dgomes** [comment]
Running 2021.8.7 since this morning without any issues ...

How are you installing rrdtool python package in the container ?

Here is what I do:

```bash
apk add build-base rrdtool-dev ttf-dejavu
pip3 install --user --no-cache-dir --prefix= --no-dependencies rrdtool
```

**Comment by ma-zal** [comment]
I installed `rrdtool` by `apk add rrdtool` (mentioned in README). But when I used your `pip` installation, then situation is same.
It look like the rrd file is not portable between architectures (related to https://stackoverflow.com/questions/52696180/convert-rrd-files-from-32-bit-arm-to-x86-64). But I have no idea, why in my Rasperry the achitecture looks like different in Docker and on host system.

So finally solved by `rrdtool dump` in host system and then `rrdtool restore` in Docker container.
